src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg, secondLeg, thirdLeg;
+    private String color;
+
+    public RightTriangle(double firstLeg, String color) {
```

Comment:
The constructor of `RightTriangle` should take two legs as parameters, not just one. A right triangle cannot have all sides equal.